locale/fof/user-bio/cs.yml

```diff
@@ -1,3 +1,3 @@
 fof-userbio:
   forum:
-    userbioPlaceholder: Write something about yourself
\ No newline at end of file
+    userbioPlaceholder: Něco nám tu o sobě napiš
```

Comment:
Napiš zde něco o sobě (je to významově to samé, ale takhle mi to přijde lepší :) )

Reply:
Možná ještě:
Představ se ostatním

Reply:
Taky by šlo, i když mi přijde, že to má trochu jiný význam.